docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -231,6 +231,65 @@ image::images/obs-ai-logs.gif[Observability AI assistant example, 75%]
 
 You can continue a conversation from a contextual prompt by clicking *Start chat* to open the AI Assistant chat.
 
+[discrete]
+[[obs-ai-connector]]
+=== AI Assistant connector
```

Comment:
Just something I noticed when browsing the page, we have a lot of headings that begin with "AI Assistant" and it makes it a little difficult to scan the page TOC. Just wondering if we could do without the AI Assistant part in this section?

![image](https://github.com/elastic/observability-docs/assets/114418652/54c65f12-c2a3-4657-b214-864d9124472f)

Reply:
Yah I that goal-oriented titles would be better. I was mainly trying to slot this into the existing structure, but it wouldn't take much work to change this.